### DISCUSSION TITLE
Raven Studio schedule Queries to export only LastModified documents

### DISCUSSION THREAD

**Discussion by kamudha4over** [ravendb/ravendb#16305] [Q&A]
Hi, I am looking for a way to schedule queries to retrieve documents that are modified the previous day for a particular collection.
I am able to query Index DocumentsByEntityName using Tag:CollectionName on the Query tab by selecting the Index but I am not sure how to schedule this query to run every day. Any ideas ?

**Comment by garayx** [comment]
the query view in the studio is to run the query manually, if you need some sort of automization you can use the RavenDB Client, and write your own logic.

https://ravendb.net/docs/article-page/5.4/csharp/client-api/creating-document-store
https://ravendb.net/features/clients/csharp

**Reply by kamudha4over** [reply]
Can you give me an example of a query using csharp to pull the documents modified from midnight to current timestamp ?

Basically, I would like to run this query every hour to generate a csv of all modified documents in a collection.

I see that the default  index DocumentsbyEntityName has a LastModified field which  I assume would be helpful.

Would you be able to send me a sample query for example Employee Collection. I come from SQL background and new to ravendb. 

Any help appreciated.

**Reply by ayende** [reply]
`DocumentsbyEntityName` is an index from version 3.5   - are you still using that?
If that is the case, please note that it is not supported.

You can do that using a index such as:

```
from u in docs
select new { 
    LastModified = u["@metadata"]["@last-modified"]
}
```

And then the query:

```
from index 'DocsByLastModified'
where LastModified > '2022-01-01'
order by LastModified
```
